src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movie: Movie,
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <>
```

Comment:
I believe you shouldn’t use React fragment if you already have a wrapper

Reply:
agree